src/snowflake/snowpark/_internal/server_connection.py

```diff
@@ -434,8 +434,14 @@ def _to_data_or_iter(
         results_cursor: SnowflakeCursor,
         to_pandas: bool = False,
         to_iter: bool = False,
-        num_statements: Optional[int] = None,
```

Comment:
Did we remove this because the param is unused?

Reply:
Yes